config/test_cases/test_branch_origin_extended_conformance_install_update.yml

```diff
@@ -72,30 +28,102 @@ extensions:
+        openshift_ansible_pkgs="atomic-openshift-utils"
+        if [[ ${ATOMIC_OPENSHIFT_UTILS_INSTALL_MINOR_VERSION} == 4 ]]
+        then
+          sudo yum erase -y ansible
+          openshift_ansible_pkgs_versions="ansible-2.2.0.0"
+          openshift_ansible_pkgs=( $( cat ./OPENSHIFT_ANSIBLE_PKGS ) )
+        fi
+        for pkg in "${openshift_ansible_pkgs[@]}"
+        do
+          openshift_ansible_pkgs_versions="${openshift_ansible_pkgs_versions} ${pkg}-${ATOMIC_OPENSHIFT_UTILS_INSTALL_VERSION}"
+        done
+        echo "=== Installing atomic-openshift-utils-${ATOMIC_OPENSHIFT_UTILS_INSTALL_VERSION} packages ==="
+        sudo yum install -y ${openshift_ansible_pkgs_versions}
     - type: "script"
-      title: "install the openshift-ansible release"
+      title: "install Ansible plugins"
+      repository: "origin"
       script: |-
-        sudo yum install -y atomic-openshift-utils
+        sudo yum install -y python-pip
+        sudo pip install junit_xml
+        sudo chmod o+rw /etc/environment
+        echo "ANSIBLE_JUNIT_DIR=$( pwd )/_output/tests/ansible_junit" >> /etc/environment
+        sudo mkdir -p /usr/share/ansible/plugins/callback
+        for plugin in 'default_with_output_lists' 'generate_junit'; do
+           wget "https://raw.githubusercontent.com/openshift/origin-ci-tool/master/oct/ansible/oct/callback_plugins/${plugin}.py"
+           sudo mv "${plugin}.py" /usr/share/ansible/plugins/callback
+        done
+        sudo sed -r -i -e 's/^#?stdout_callback.*/stdout_callback = default_with_output_lists/' -e 's/^#?callback_whitelist.*/callback_whitelist = generate_junit/' /etc/ansible/ansible.cfg
     - type: "script"
-      title: "check out the correct branch of aos-cd-jobs"
+      title: "install origin GA and update"
       repository: "aos-cd-jobs"
       script: |-
-        git checkout sjb
+        pkg_name=$( cat ./PKG_NAME )
+        echo $pkg_name > DEPLOYMENT_TYPE
+        if [ $pkg_name = "atomic-openshift" ]
+        then
+          echo "openshift-enterprise" > DEPLOYMENT_TYPE
+          sed -i "s/deployment_type:.*/deployment_type: \"openshift-enterprise\" /g" inventory/group_vars/OSEv3/general.yml
+        fi
+        sudo python hack/determine_install_upgrade_version.py $( cat ORIGIN_BUILT_VERSION )
+        source ORIGIN_INSTALL_UPGRADE_VERSION
+        source ATOMIC-OPENSHIFT-UTILS_INSTALL_UPGRADE_VERSION
+        echo "=== Installing ${pkg_name}-${ORIGIN_INSTALL_VERSION} ==="
+        ansible-playbook -vv --become --become-user root --connection local -i inventory/ /usr/share/ansible/openshift-ansible/playbooks/byo/openshift-node/network_manager.yml -e deployment_type=$( cat ./DEPLOYMENT_TYPE)
+        ansible-playbook -vv --become --become-user root --connection local -i inventory/ /usr/share/ansible/openshift-ansible/playbooks/byo/config.yml -e openshift_pkg_version="-${ORIGIN_INSTALL_VERSION}" -e enable_excluders=false -e deployment_type=$( cat ./DEPLOYMENT_TYPE)
```

Comment:
The excluders are fixed and now they are installed and enabled upon the install process. During the upgrade process they are disabled so no need to disable them now